examples/next/next.config.mjs

```diff
@@ -0,0 +1,27 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```

Comment:
Is this fine, or too convoluted?

```js
// next.config.mjs

// Api keys
const {
  ALCHEMY_API_KEY,
  ETHERSCAN_API_KEY,
  ARBISCAN_API_KEY,
  BASESCAN_API_KEY,
  OPTIMISTIC_ETHERSCAN_API_KEY,
  POLYGONSCAN_API_KEY,
  ZORA_SUPERSCAN_API_KEY,
} = process.env;

const env = {
  ALCHEMY_API_KEY,
  ETHERSCAN_API_KEY,
  ARBISCAN_API_KEY,
  BASESCAN_API_KEY,
  OPTIMISTIC_ETHERSCAN_API_KEY,
  POLYGONSCAN_API_KEY,
  ZORA_SUPERSCAN_API_KEY,
};

Object.keys(env).forEach((key) => {
  if (!env[key]) {
    console.warn(
      `No ${key} environment variable set. You may face throttling.`,
    );
  }
});

/** @type {import('next').NextConfig} */
const nextConfig = {
  ...
  // These will get exposed to the browser
  // If you would like to keep this key private, use an API route/server components to
  // access it and return the data instead
  env,
};

export default nextConfig;
```

Reply:
It's fine as long as destructuring process.env like this doesn't cause issues. for most frameworks you need to reference process.env.FOO directly because webpack or vite replace the environment variables in place (process.env doesn't exist in the browser)